tests/tests.py

```diff
@@ -216,6 +217,42 @@ def test_success_all_fields(self):
         self.assertEqual(prediction.classifications[1].prediction, "color")
 
 
+class TestExtract(unittest.TestCase):
+    def test_success(self):
+        extract = Extract(
+                examples=[ExtractExample(
+                    text="hello my name is John, and I like to play ping pong", 
+                    entities=[ExtractEntity(type="Name", value="John")])],
+                texts=["hello Roberta, how are you doing today?"])
+
+        extractions = co.extract(
+            'small', extract)
+
+        self.assertIsInstance(extractions, list)
+        self.assertIsInstance(extractions[0].text, str)
+        self.assertIsInstance(extractions[0].entities, list)
+        self.assertEqual(extractions[0].entities[0].type, "Name")
+        self.assertEqual(extractions[0].entities[0].value, "Roberta")
+
```

Comment:
add more tests
1. multiple example, each with different amount of extractentity
2. multiple example with multiple texts
3. multiple examples with multiple text but one of the text doesnt contain any extractentities